apache/setup.py

```diff
@@ -78,7 +53,7 @@ def find_version(*file_paths):
     author_email='packages@datadoghq.com',
 
     # License
-    license='MIT',
+    license='New BSD',
```

Comment:
just `BSD`

Reply:
tbh neither of them are really correct it's bsd 3 clause but we should keep it standard